MKOB/MKOB.pyw

```diff
@@ -24,19 +24,25 @@ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 SOFTWARE.
 """
 
-"""MKOB.pyw
+"""
+MKOB.pyw
 
 Python version of MorseKOB 2.5
+
+MorseKOB 4.0.1  2020-06-10
+- fix jitter in code reader window
```

Comment:
I'm fine with this for now, but at some point the 'Version Info' should be auto generated into a 'versions.md' (or similar) file, not manually included into the source file.

Reply:
Ah, you beat me to it! I was planning to raise the whole question of version control. I need to be educated on best practices in this area.